editions/tw5.com/tiddlers/howtos/How to export tiddlers.tid

```diff
@@ -0,0 +1,30 @@
+created: 20151006145934884
+modified: 20151006155211988
+tags: Learning
```

Comment:
I'd be for putting it under "Working with TiddlyWiki" ...and just call it "Exporting Tiddlers"... but then there is a tiddler called "ImportTiddlers", so not sure if we should use the same naming convention for both.